src/com/google/enterprise/adaptor/MetadataTransform.java

```diff
@@ -25,34 +27,57 @@
  */
 public interface MetadataTransform {
 
-  /** Keys for data stored in the {@code Map} of {@code params}. */
+  /** The key for the document's unique ID supplied in the {@code params}. */
   public static final String KEY_DOC_ID = "DocId";
+
+  /** The key for the document's display URL supplied in the {@code params}. */
   public static final String KEY_DISPLAY_URL = "Display-URL";
+
+  /** The key for the document's content type supplied in the {@code params}. */
   public static final String KEY_CONTENT_TYPE = "Content-Type";
+
+  /** The key for the document's crawl once flag supplied in the {@code params}.
+   */
   public static final String KEY_CRAWL_ONCE = "Crawl-Once";
+
+  /** The key for the document's lock flag supplied in the {@code params}. */
   public static final String KEY_LOCK = "Lock";
+
+  /** The key for the document's last modified timestamp supplied in the
+   * {@code params}.
+   */
   public static final String KEY_LAST_MODIFIED_MILLIS_UTC
       = "Last-Modified-Millis-UTC";
+
+  /** The key for a {@link TransmissionDecision} that may be added to the
+   * {@code params}.
+   */
   public static final String KEY_TRANSMISSION_DECISION 
       = "Transmission-Decision";
 
   /**
    * Identifies the location a configured key/value pair resides,
-   * either in the {@link Metadata}, or the map of {@code params}.
+   * either in the {@link Metadata}, or the map of {@code params}
+   * supplied to the {@link #transform} method of a {@code MetadataTransform}.
    * <p>
-   * Suppose a transform wishes examine or modify the display URL, which is
-   * stored in the {@code params}. The transform's configuration might look
-   * like:
+   * Suppose a transform wishes to examine or modify the display URL, which is
+   * stored in the {@code params}. The transform's configuration might include:
    * <pre><code>
      key=Display-URL
      keyset=params
      </code></pre>
-   * If unspecified, the default value is {@code metadata}.
+   * If unspecified, the default value is {@code METADATA}.
```

Comment:
I don't like this change, since in this context, it's the name that matters, not the enum identifier.